edisgo/io/db.py

```diff
@@ -169,9 +175,27 @@ def engine(path: Path | str = None, ssh: bool = False) -> Engine:
     """
 
     if not ssh:
+        # Github Actions KHs token
+        if "TOEP_TOKEN_KH" in os.environ:
+            env_var = os.environ["TOEP_TOKEN_KH"]
+        else:
+            try:
+                with open(token) as file:
```

Comment:
What should the default location for the token be? The config directory? It should be added